    public function testThrowsExceptionIfNotTraversable()
    {
        $this->setExpectedException('Symfony\Component\Validator\Exception\UnexpectedTypeException');

        $this->validator->isValid('foobar', new All(new Min(4)));
    }

    /**
     * @dataProvider getValidArguments
     */
||||||||    public function testThrowsExceptionIfNotTraversable()
    {
        $this->validator->isValid('foo.barbar', new All(new Min(4)));
    }

    /**
     * @dataProvider getValidArguments
     */
||||||||NO_DOC_COMMENTNO_RETURN_TYPE
(
    (AST_METHOD_CALL
        (AST_VAR)
        (
            (SCALAR)))
    (AST_METHOD_CALL
        (AST_PROP
            (AST_VAR))
        (
            (SCALAR)
            (AST_NEW
                (
                    (AST_NEW
                        (
                            (SCALAR))))))))||||||||HAS_DOC_COMMENTNO_RETURN_TYPE
(
    (AST_METHOD_CALL
        (AST_PROP
            (AST_VAR))
        (
            (SCALAR)
            (AST_NEW
                (
                    (AST_NEW
                        (
                            (SCALAR))))))))